content/topics/work/mock-interview-survey/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+ready: true
+title: Mock Interviews Survey
+---
+
+### Mock Interview Survey
```

Comment:
Repetitive headings. 
Repetitive headings

Please. I keep giving you the same feedback over and over

Focus. Jiust look at the stuff and you will see